src/assets/content.tsx

```diff
@@ -146,79 +140,37 @@ const content: Content = {
   whatsNew: {
     title: 'Was ist neu?',
     description: [
-      `Wir haben die Farben angepasst, damit der Wasserbedarf der Berliner Bäume intuitiver zu erfassen ist. Es beginnt bei einem trockenem Gelb und geht bis in ein sattes Grün.<br /><div style="padding-top:0.5rem;padding-bottom:0.5rem; display:flex">
-      <div
-        style=" background-color:#fde725; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#fde725"
-      ></div>
-      <div
-        style=" background-color:#cae11f; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#cae11f"
-      ></div>
-      <div
-        style=" background-color:#95d840; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#95d840"
-      ></div>
-      <div
-        style=" background-color:#63cb5f; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#63cb5f"
-      ></div>
-      <div
-        style=" background-color:#3bbb75; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#3bbb75"
-      ></div>
-      <div
-        style=" background-color:#22a884; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#22a884"
-      ></div>
-    </div> Ebenfalls haben wir unsere Pumpen-Daten geupdated. Die kommen jetzt jede Woche frisch von Open Street Maps. `,
-      `Ihr könnt sehen ob Pumpen: "funktionsfähig" <span
-       style=
-       "display: inline-block;
-        background-color: ${workingColor.hex};
-        width: 13px;
-        height: 13px;
-        border-radius: 100px;
-        margin-right: 5px;"
+      `Der Frühling kann kommen! Wir haben unser Backlog aufgeräumt, einige Funktionen verbessert und neue Funktionen hinzugefügt. Die wichtigsten Verbesserungen im Überblick:<br />
+      <div style="padding-top:0.5rem;padding-bottom:0.5rem; display:flex">
+        <div>
+          <img style="max-width:18px; margin-right: 5px;" alt="Geolocate Icon" src="/images/geolocate-icon.png"/><b>Geolokalisierung</b><br>
```

Comment:
The icons should have the same height as the cap height of the text

Reply:
fixed with [this commit](https://github.com/technologiestiftung/giessdenkiez-de/pull/262/commits/fbfced23979a9940327ec6208136aacba7b6205c)